src/unix/mod.rs

```diff
@@ -298,6 +299,7 @@ cfg_if! {
     } else if #[cfg(feature = "std")] {
         // cargo build, don't pull in anything extra as the libstd dep
         // already pulls in all libs.
+        extern {}
```

Comment:
Could you clarify why you've added this?

Reply:
Ah, sorry, it's a leftover from one of the experiments, https://github.com/rust-lang/libc/pull/2037/commits/355301417300e33fd48571f4c0248c7f5db9254c. Will remove this in a second.